plugins/crop_generators/sampled_crop_generator.py

```diff
@@ -115,20 +115,21 @@ def generator(self, sess, global_step=0):
             for i in range(len(batch_paths)):
                 x[i, :, :, :] = images[i]
 
-            yield (x, tf.keras.utils.to_categorical(y, num_classes=self.num_classes))
+            yield x, tf.keras.utils.to_categorical(y, num_classes=self.num_classes)
 
         image_loader.close()
 
-    def generate(self):
+    def generate(self, sess):
```

Comment:
Same comment here. Is `sess` necessary? If it is part of the interface for compatibility, that sounds reasonable to me. Just checking.

Reply:
It is a part of interface compatibility.